lib/blueprintLocations.js

```diff
@@ -5,40 +5,40 @@ module.exports = function($) {
 
     const tbody = table.children()['0']
 
-    let blueprint = null
+    let item = null
 
-    let blueprints = []
+    let items = []
 
     for(let tr of tbody.children) {
         let elem = tr.children[0]
         let text = $(elem).text()
 
         if(elem.name === "th" && tr.children.length === 1) {
-            if(blueprint) {
-                blueprints.push(blueprint)
+            if(item) {
+                items.push(item)
             }
 
-            blueprint = {_id: hash(text), blueprintName: text, enemies: []}
+            item = {_id: hash(text), itemName: text, enemies: []}
         }
 
         if(elem.name === "td" && elem.attribs.class !== "blank-row") {
             let chanceElem = tr.children[2]
             let chance = parseChance($(chanceElem).text())
 
-            let enemyBlueprintDropChance = $(tr.children[1]).text()
+            let itemdropchance = $(tr.children[1]).text()
 
-            blueprint.enemies.push({
+            item.enemies.push({
                 _id: hash(text),
                 enemyName: text,
-                enemyBlueprintDropChance: Number(enemyBlueprintDropChance.slice(0, enemyBlueprintDropChance.length - 1)),
+                enemyItemDropChance: Number(itemdropchance.slice(0, itemdropchance.length - 1)),
                 rarity: chance.rarity,
                 chance: Number(chance.chance)
             })
         }
     }
 
     // push the last one too
-    blueprints.push(blueprint)
+    items.push(item)
 
-    return blueprints
-}
\ No newline at end of file
+    return items
```

Comment:
please maintain passivity, this can break a lot of people's code with little to no notification or time to fix beforehand.